### DISCUSSION TITLE
DDEV Installation - Windows WSL2

### DISCUSSION THREAD

**Discussion by addison74** [ddev/ddev#4246] [Q&A] [answered]
Please visit this documentation page https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/ and choose **Windows WSL2**.

At step 14 it says to run the following command to allow ```ddev launch``` to work:

```
sudo apt-get update && sudo apt-get install -y certutil xdg-utils to install the xdg-utils
```
Because it gave me an error I looked for the ```certutil``` package on the official Ubuntu page. As can be seen from the image below this package does not exist.

![screenshot](https://user-images.githubusercontent.com/8360474/193451805-dd38ece6-a69f-4e38-9344-602b98c269fe.png)

I personally think that a modification of the existing documentation is mandatory by using this package ```libnss3-tools``` which contains **certutil** command.

**Comment by rfay** [comment, marked as answer]
Well, your contribution is always welcome, and contributing to the docs is really easy. 

On any docs page, look for the pencil icon on the upper right and click it, and create a docs PR. It's a lot easier than making an issue or a discussion page.

Thanks!

**Reply by rfay** [reply]
One note: If you think something is wrong with DDEV or the docs, create an issue or a PR. These discussions are more for things that should be discussed, not for things like this that need to be addressed for sure.

**Reply by addison74** [reply]
It won't be a problem based on your recommendation, but if you take a look at the two links you will notice that there are differences. The one where I identified the issue has 15 steps and the one in GitHub has 14 steps. Step 14 is missing, which was removed according to the history on August 22. I leave it to your discretion whether that step should be added, but without it we cannot launch ```ddev launch``` command in the Ubuntu container.

**Reply by rfay** [reply]
Just make the PR and if there's some reason to disagree with it it will come up in the PR.

**Reply by rfay** [reply]
Since you're using it, your opinion matters a LOT!

**Reply by addison74** [reply]
In the OpenMage project where I have been contributing for some time, DDEV was discussed in August. I didn't know about it, but after almost a month and a half of turning it all over in Windows, I think it's an option to consider it for development/testing for those who either use XAMPP/WAMP or a virtual machine. I also followed your YouTube channel and I appreciate your work, especially since it serves everything related to Docker configuration. I think many have told you, so I won't insist any more. I am going to create a PR to discuss how to use that command, from what I have noticed it is been asked about recently.

**Reply by rfay** [reply]
It's lovely to have an OpenMage expert in the neighborhood too. You might consider joining us on [discord](https://discord.gg/hCZFfAMc5k) as well.
